src/__experimental__/components/radio-button/radio-button-svg.component.js

```diff
@@ -26,4 +27,4 @@ const RadioButtonSvg = () => {
   );
 };
 
-export default RadioButtonSvg;
+export default React.memo(RadioButtonSvg, () => true);
```

Comment:
This actually does nothing.
`() => true` means that this component will behave the same as if React.memo has not been used.

Reply:
It turned out I'm wrong
https://reactjs.org/docs/react-api.html#reactmemo

Reply:
Resolved in Slack.

`() => true` means to always re-use the same component.